include/deal.II/lac/petsc_ts.h

```diff
@@ -81,8 +81,8 @@ namespace PETScWrappers
      */
     TimeStepperData(
       // Running parameters
-      const std::string &opts_prefix       = "",
-      const std::string &tstype            = "",
+      const std::string &options_prefix    = "",
```

Comment:
PETScWrappers::SolverBase uses prefix_name for options prefix. Here and in SNES, we use `options_prefix`. Should we make them uniform and use prefix_name to preserve old codes that are already using SolverBase? @bangerth @luca-heltai

Reply:
I'll take a look at what you are doing in the other deal.II PETSc classes and decide.